content/en/news/releases/1.20.x/announcing-1.20/change-notes/index.md

```diff
@@ -0,0 +1,228 @@
+  ([Issue #47341](https://github.com/istio/istio/issues/47341))
+
+- **Fixed** an issue where using a `Sidecar` resource with `IstioIngressListener.defaultEndpoint` could not use [::1]:PORT
+  if the default IP addressing was not IPv6.
+  ([Issue #47412](https://github.com/istio/istio/issues/47412))
+
+- **Fixed** multicluster secret filtering causing Istio to pick up secrets from every namespace.
+  ([Issue #47433](https://github.com/istio/istio/issues/47433))
+
+- **Fixed** an issue causing traffic to terminating headless service instances to not function correctly.
+  ([Issue #47348](https://github.com/istio/istio/issues/47348))
+
+- **Removed** the `PILOT_ENABLE_DESTINATION_RULE_INHERITANCE` experimental feature, which has been disabled by default
+  since it was created.
+  ([Issue #37095](https://github.com/istio/istio/issues/37095))
+
+- **Removed** custom Istio network filters `forward_downstream_sni`, `tcp_cluster_rewrite`, and `sni_verifier` from
+  the Envoy build. This functionality can be achieved using Wasm extensibility.
+
+- **Removed** the requirement for a workload to have a `Service` associated with it for locality load balancing
+  to work.
+
+## Security
+
+- **Added** the capability to attach `RequestAuthentication` to Kubernetes `Gateway` resources via the `targetRef`
+  field.
+
+- **Added** support for plugged root cert rotation.
+
+- **Fixed** an issue where all requests were being denied when the custom external authorization service had an issue.
+  Now only requests that are delegated to the custom external authorization service are denied.
+  ([Issue #46951](https://github.com/istio/istio/issues/46951))
+
+## Telemetry
+
+- **Added** the capability to attach `Telemetry` to Kubernetes `Gateway` resources via the `targetRef`
+  field. ([Issue #46844](https://github.com/istio/istio/issues/46844))
+
+- **Added** xDS workload metadata discovery to the TCP metadata exchange filter as a fallback. This requires
+  enabling the `PEER_METADATA_DISCOVERY` flag on the proxy and `PILOT_ENABLE_AMBIENT_CONTROLLERS` on the control plane.
+
+- **Added** a flag `PILOT_DISABLE_MX_ALPN` on the control plane to disable advertising the TCP metadata exchange ALPN
+  token `istio-peer-exchange`.
+
+## Extensibility
+
+- **Added** the capability to attach `WasmPlugin` to Kubernetes `Gateway` resources via the `targetRef` field.
+
+## Installation
+
+- **Improved** Usage on OpenShift clusters by removing the need to grant the `anyuid` SCC privilege to
+  Istio and applications.
+
+- **Updated** the Kiali addon to version `v1.76.0`.
+
+- **Added** `volumes` and `volumeMounts` to the gateways chart.
+
+- **Added** basic ztunnel support for revisions when installing with `istioctl`.
+  ([Issue #46421](https://github.com/istio/istio/issues/46421))
+
+- **Added** an env var `PILOT_ENABLE_GATEWAY_API_GATEWAYCLASS_CONTROLLER` to enable/disable management of built-in
+  `GatewayClasses`.
```

Comment:
```suggestion
- **Added** flag `PILOT_ENABLE_GATEWAY_API_GATEWAYCLASS_CONTROLLER` to enable/disable management of built-in
  `GatewayClasses`.
```

We refer to these as flags elsewhere in the change notes, so mostly for consistency